src/components/board/Board.jsx

```diff
@@ -223,6 +224,7 @@ export function Board() {
                     groupId={group.id}
                     title={group.title}
                     tasks={tasks}
+                    board={board}
```

Comment:
you dont need to pass board here.. is prop drilling. thats why we use redux, get the board in column component from the store.